backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -120,13 +120,13 @@ public void testGetTokensForShop_success() throws Exception {
 
         // Assert.
         assertEquals("Size of token list", getTokensResponseList.size(),
-                inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().get(shopId).size());
+                inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().get(shopId).stream().filter(token -> token.getStatus() == Status.ACTIVE).count());
```

Comment:
Reverse the order here and in other tests applicable?
expected value and then actual value that we got in the act block?

Reply:
done!